monzo.js

```diff
@@ -1,20 +1,83 @@
 var PD = require("probability-distributions");
 
-var goal = 25000;
-var pledgeValues = [10,20,50,100,250,500,1000];
+var goal = 2500;
+var pledgeValues = [10,20,50,100];
 var maxPledgeValue = pledgeValues[pledgeValues.length - 1];
-var dataset = PD.sample(pledgeValues, 1000, true, [7,6,5,4,3,2,1]);
+var dataset = PD.sample(pledgeValues, 1000, true, [7,6,5,4]);
 
 var datasetTotal = dataset.reduce((a, b) => a + b, 0);
```

Comment:
ES6 only, you need a solution to this if the script is going to be used in a browser that doesn't support it